packages/vega3-extension/src/index.ts

```diff
@@ -84,10 +84,27 @@ class RenderedVega3 extends Widget implements IRenderMime.IRenderer {
           loader,
           actions: true
         };
-        return vegaEmbed(this.node as HTMLBaseElement, data, options).then((result) => {
-          // result contains the Vega spec and view
+        return vegaEmbed(this.node as HTMLBaseElement, data, options).then((result: any) => {
```

Comment:
Why is any necessary here?

Reply:
That was only necessary to get this to build with vega-embed 3.1.1 because `result` has `{}` type. I'll remove with the next commit.

Reply:
Now do you mean that you'll remove it when we upgrade to 3.4.0 in a separate PR?